webpack/JobInvocationDetail/index.js

```diff
@@ -120,6 +121,7 @@ const JobInvocationDetailPage = ({
           />
         </Flex>
       </Flex>
+      {items.id !== undefined && <JobInvocationHostTable id={id} />}
```

Comment:
Not sure if its in the scope of the PR, but previously we had this:
```
  <div class="alert alert-warning">
    <%=
        _("The dynamic query '%{query}' was not resolved yet. The list of hosts to which it would resolve now can be seen %{here}." %
            { :query => job_invocation.targeting.search_query,
              :here => link_to(_('here'), hosts_url(:search => job_invocation.targeting.search_query))})
     %>
  </div>
```
if the table was empty

Reply:
The empty table in the old UI looks like this:
![image](https://github.com/user-attachments/assets/6ed9c071-83a7-4a42-95ef-2e24b019c39d)

I made it to look like this now:
![image](https://github.com/user-attachments/assets/49663ac5-4383-463c-913d-698619564a1e)

I don't know when the message from the code you sent appears. :thinking:

Reply:
Looks good! the error is only when `job_invocation.resolved?` (I missed that)

Reply:
I'm not sure what you mean. This empty table happens when  `!isPending && results.length === 0 && !errorMessage`.

Reply:
When a user starts a job in the future, and has a dynamic hosts list, there will be no hosts until the job starts.
So when that happens in the old page an alert will show up.

Reply:
Thank you for explaining it, @asteflova helped me with rewording it and I incorporated it like this:
![image](https://github.com/user-attachments/assets/08483751-4d0e-4972-b146-b59f78c2a7e2)